data/contact-info.json

```diff
@@ -7,18 +7,24 @@
     "imageURI":"../data/images/contacts/pubsafe.jpg"
   },
   {
+    "title":"SARN",
+    "text":"SARN, the Sexual Assault Resource Network, is a confidential resource availiable to all students. Calls to SARN will be picked up by a trained advocate to provide services to students struggling with issues of sexual and domestic violence. Students can also visit the SARN office in Buntrock Commons room 113.",
+    "phoneNumber":"5077863777",
+    "buttonText":"Call SARN",
+    "imageURI":"../data/images/contacts/pause.jpg"
```

Comment:
The intention was to move this up in the list, per a request from Emma Lind after I demoed it for her today. Is that what you're referring to or am I missing something?